### PR TITLE
Base room, plus other stuff

### PR DESCRIPTION
**Pull request by Juhoton**
FIrst version of the Base room. Made in a new scene, and a made it into a new prefab, so that I can continue the editing of the room, and it should make those changes to other people using the room. Also changed the location of wall related items to the building folder. Changed some wall names.

### CODE REVIEW

File: Assets/Materials/Television_var.mat [hunk 1]
```diff
@@ -26,5 +26,5 @@ Material:
     m_Ints: []
     m_Floats: []
     m_Colors:
-    - _Color: {r: 0.3058179, g: 0.3058179, b: 0.3058179, a: 0}
+    - _Color: {r: 0.30581748, g: 0.30581748, b: 0.30581748, a: 0}
```
Comment (viilis) [inline Assets/Materials/Television_var.mat:29]: unnecessary change

File: Assets/Models/Building/RightCornerWall4 - copy.fbx
Comment (viilis) [inline Assets/Models/Building/RightCornerWall4 - copy.fbx:1]: Why there are copies of models?

File: Assets/Models/Building/WallAtlas - Copy.png
Comment (viilis) [inline Assets/Models/Building/WallAtlas - Copy.png:1]: same here

File: Assets/Models/Building/Wall4 - copy.fbx
Comment (viilis) [inline Assets/Models/Building/Wall4 - copy.fbx:1]: copy here

File: Assets/Models/Building/LeftCornerWall3 - copy.fbx
Comment (viilis) [inline Assets/Models/Building/LeftCornerWall3 - copy.fbx:1]: copy?

**Review by viilis — CHANGES_REQUESTED** [review]

File: Assets/Materials/Television_var.mat (hunk 1 above)
Comment (Juhoton) [reply Assets/Materials/Television_var.mat:29]: Yeah, I don't actually know why that was changed. If I remember correctly, I didn't touch the asset.

File: Assets/Models/Building/RightCornerWall4 - copy.fbx
Comment (Juhoton) [reply Assets/Models/Building/RightCornerWall4 - copy.fbx:1]: Found similar models, so named the extra ones into copies, rather than deleting them

**Comment by Juhoton** [comment]
Removed the copies. Changed the movable_player camera height to 0.6 from 0.87. So currently it should be around 1.6 meters, which is closer to where the average eyes actually are. Also dropped the fov to 70, have to check around what feels best and creates better immersion in game. Television_var is fucked.

**Review by viilis — APPROVED** [review]
